js_modules/dagster-ui/packages/ui-core/src/app/ContentRoot.tsx

```diff
@@ -35,97 +36,101 @@ export const ContentRoot = memo(() => {
       <ErrorBoundary region="page" resetErrorOnChange={[pathname]}>
         <Switch>
           <Route path="/asset-groups(/?.*)">
-            <Suspense fallback={<div />}>
+            <TrackedSuspense fallback={<div />} id={id('asset-groups')}>
               <AssetsGroupsGlobalGraphRoot />
-            </Suspense>
+            </TrackedSuspense>
           </Route>
           <Route path="/assets(/?.*)">
-            <Suspense fallback={<div />}>
+            <TrackedSuspense fallback={<div />} id={id('assets')}>
```

Comment:
I have two questions here:

1) Why is it necessary to put the Suspense inside the Route and not inside the Switch on line 37? Does this really need to be repeated for every route separately like this? Is this primarily so the `id` can be passed? 

2) If the reason for #1 is so that the suspense instance is replaced when you navigate, do these `/*` type routes all share the same tracked suspense? Is that ok?

Reply:
It's not necessary, I was just replacing the existing `Suspense` calls with `TrackedSuspense` in a follow up we can consolidate these but didn't want to unknowingly break anything by doing that